paddle/fluid/inference/api/analysis_predictor.h

```diff
@@ -436,6 +447,12 @@ class AnalysisPredictor : public PaddlePredictor {
 
   std::map<std::string, std::vector<std::vector<int32_t>>> shape_info_;
   int clone_num_{1};
+
+  // fleet executor related
+  distributed::FleetExecutorDesc executor_desc_;
+  std::shared_ptr<distributed::FleetExecutor> fleet_exe_;
+  std::shared_ptr<distributed::TaskNode> task_node_;
+  std::vector<std::string> force_root_scope_var_names_{};
```

Comment:
Better to add Macro Variable to control whther to compile with `FleetExecutor`, so that the inference library size is not affected.

Reply:
oh yeah, i forgot those compile flags, will add soon~